docs/security-considerations.rst

```diff
@@ -28,9 +28,16 @@ As always, with open source documentation, please help us extend this section
 (especially, some examples would not hurt)!
 
 ********
-Pitfalls
+Pitfalls and Vulnerabilities
```

Comment:
I am not sure if the word "vulnerability" is the right one in this context. If there is a vulnerability in Solidity, I would interpret that as something where I can trick the compiler to do something it would not normally do.